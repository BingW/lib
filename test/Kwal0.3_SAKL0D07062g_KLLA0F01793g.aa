	3	134
>Kwal0.3
MTTGLMRHFSRTKAHRDSMFRNLVSQLFQHGSIVSTHEKCLEASRMAERIITMAKEIQSRLSLAGDNSKLLRRVVNELAPRYQQRPGGYTRVLKLEPRLGDRAKQSVLELVDAPIVGENGTLQRGDLKLWLLCK
>SAKL0D07062g
MTVGLARHLSRTKPHRDALLKNLVTQLFQHGSITSTHEKCKEASRLAERAITIAKELQSRLFHSGDNSKVLKKLLQEIAPRCVTRPGGYTRVLHLEPRLGDRARQSVLELVDNPVIDQNGQFQRGNLKLWLLCK
>KLLA0F01793g
MTRGLARHLSRTKPHRDALIRNMATQLLQHGTISSTtpKLKETQRYVERIITIAKELQSRLYLSGDNSKLLKKLFDEIAPRYTARNGGYTRVMKLEPRLGDRAPQAILELVDAPVQNEDGQLLKGNLKLWLLTK
