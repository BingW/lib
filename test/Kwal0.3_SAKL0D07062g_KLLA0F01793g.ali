 
                           DIALIGN 2.2.1 
                           *************
 
          Program code written by Burkhard Morgenstern and Said Abdeddaim 
             e-mail contact: dialign (at) gobics (dot) de 
 
          Published research assisted by DIALIGN 2 should cite:  
 
             Burkhard Morgenstern (1999).
             DIALIGN 2: improvement of the segment-to-segment
             approach to multiple sequence alignment.
             Bioinformatics 15, 211 - 218. 

          For more information, please visit the DIALIGN home page at 

             http://bibiserv.techfak.uni-bielefeld.de/dialign/ 
 
         ************************************************************
 


   program call:  dialign2-2 -fa /Users/bingwang/VimWork/new_output/Kwal_Sklu_Klac/Kwal0.3_SAKL0D07062g_KLLA0F01793g.fasta  

 
   Aligned sequences:          length:
   ==================          =======
 
   1) Kwal0.3                    239
   2) SAKL0D07062g               241
   3) KLLA0F01793g               246

   Average seq. length:          242.0 


   Please note that only upper-case letters are considered to be aligned. 


   Alignment (DIALIGN format):
   ===========================
 
Kwal0.3            1   MTTGLMRHFS RTKAHRDSMF RNLVSQLFQH GSIVSTHEKC LEASRMAERI 
SAKL0D07062g       1   MTVGLARHLS RTKPHRDALL KNLVTQLFQH GSITSTHEKC KEASRLAERA 
KLLA0F01793g       1   MTRGLARHLS RTKPHRDALI RNMATQLLQH GTISSTtpKL KETQRYVERI 
                                                                 
                       8888888888 8888888888 8888888888 8888883344 0111111111 
 
Kwal0.3           51   ITMAKKLEQH PNsar---EI QSRLSLAGDN SKLLRRVVNE LAPRYQQRPG 
SAKL0D07062g      51   ITIAKRYNSN PDvki-VSEL QSRLFHSGDN SKVLKKLLQE IAPRCVTRPG 
KLLA0F01793g      51   ITIAKNGIQN GNpsssVPEL QSRLYLSGDN SKLLKKLFDE IAPRYTARNG 
                                                                 
                       1111111111 1100002266 6666666666 6666666666 6666555888 
 
Kwal0.3           98   GYTRVLKLEP RLGDRAKQSV LELVDAPIVG ENGTLQRGDL KLWLLCKAAI 
SAKL0D07062g     100   GYTRVLHLEP RLGDRARQSV LELVDNPVID QNGQFQRGNL KLWLLCKTML 
KLLA0F01793g     101   GYTRVMKLEP RLGDRAPQAI LELVDAPVQN EDGQLLKGNL KLWLLTKTcm 
                                                                 
                       8888885699 9999999999 9999999999 9995588888 8888888522 
 
Kwal0.3          148   QNEASQEAYS RLTLQNFAKM LKFRpdphAF FVNDVMAVKK FI-------- 
SAKL0D07062g     150   ANESAQEPYS DNTLQNFRKI AKFQndfdAF LSKDVLAIRK FI-------- 
KLLA0F01793g     151   qqiedagqla pkTVQNLKKM Vhnktldq-- LLHEIGVIRK YIlesqqqns 
                                                                 
                       2222222222 2222222222 2222000011 1111111111 1100000000 
 
Kwal0.3          190   --KQEQGLEW DEKHEIGLLE QLESQVLGLS AQPEtsqiQK A---GYTFVN 
SAKL0D07062g     192   --KLQEQQPW DEEQELQQLK VLKQKVLSPS QKKN----Hs gstpGFNYVS 
KLLA0F01793g     199   ttEVSELPEF DETLQQQFLA AVKDQIQNHT AEKK----QP D---GYVFr- 
                                                                 
                       0011111111 1111111111 1111111000 0000000000 0000000000 
 
Kwal0.3          235   ERPGR- 
SAKL0D07062g     236   KRPERl 
KLLA0F01793g     241   ERPtqg 
                 
                       000000 
 

 
 
   Sequence tree:
   ==============

Tree constructed using UPGMAbased on DIALIGN fragment weight scores
 
((Kwal0.3     :0.005858SAKL0D07062g:0.005858):0.002575KLLA0F01793g:0.008434);
 
 
