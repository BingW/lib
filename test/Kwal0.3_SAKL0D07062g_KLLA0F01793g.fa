>Kwal0.3
MTTGLMRHFSRTKAHRDSMFRNLVSQLFQHGSIVSTHEKCLEASRMAERI
ITMAKKLEQHPNsar---EIQSRLSLAGDNSKLLRRVVNELAPRYQQRPG
GYTRVLKLEPRLGDRAKQSVLELVDAPIVGENGTLQRGDLKLWLLCKAAI
QNEASQEAYSRLTLQNFAKMLKFRpdphAFFVNDVMAVKKFI--------
--KQEQGLEWDEKHEIGLLEQLESQVLGLSAQPEtsqiQKA---GYTFVN
ERPGR-
 
>SAKL0D07062g
MTVGLARHLSRTKPHRDALLKNLVTQLFQHGSITSTHEKCKEASRLAERA
ITIAKRYNSNPDvki-VSELQSRLFHSGDNSKVLKKLLQEIAPRCVTRPG
GYTRVLHLEPRLGDRARQSVLELVDNPVIDQNGQFQRGNLKLWLLCKTML
ANESAQEPYSDNTLQNFRKIAKFQndfdAFLSKDVLAIRKFI--------
--KLQEQQPWDEEQELQQLKVLKQKVLSPSQKKN----HsgstpGFNYVS
KRPERl
 
>KLLA0F01793g
MTRGLARHLSRTKPHRDALIRNMATQLLQHGTISSTtpKLKETQRYVERI
ITIAKNGIQNGNpsssVPELQSRLYLSGDNSKLLKKLFDEIAPRYTARNG
GYTRVMKLEPRLGDRAPQAILELVDAPVQNEDGQLLKGNLKLWLLTKTcm
qqiedagqlapkTVQNLKKMVhnktldq--LLHEIGVIRKYIlesqqqns
ttEVSELPEFDETLQQQFLAAVKDQIQNHTAEKK----QPD---GYVFr-
ERPtqg
 