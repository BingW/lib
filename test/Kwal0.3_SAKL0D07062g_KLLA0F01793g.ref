Kwal0.3             MTTGLMRHFSRTKAHRDSMFRNLVSQLFQHGSIVSTHEKCLEASRMAERIITMAKKLEQH
SAKL0D07062g        MTVGLARHLSRTKPHRDALLKNLVTQLFQHGSITSTHEKCKEASRLAERAITIAKRYNSN
KLLA0F01793g        MTRGLARHLSRTKPHRDALIRNMATQLLQHGTISSTtpKLKETQRYVERIITIAKNGIQN
gene_mark           :: :: :: :::: :::    :   :: ::: : ::  :  :  :  :: :: ::     
gene_keep           =======================================================     

Kwal0.3             PNsar---EIQSRLSLAGDNSKLLRRVVNELAPRYQQRPGGYTRVLKLEPRLGDRAKQSV
SAKL0D07062g        PDvki-VSELQSRLFHSGDNSKVLKKLLQEIAPRCVTRPGGYTRVLHLEPRLGDRARQSV
KLLA0F01793g        GNpsssVPELQSRLYLSGDNSKLLKKLFDEIAPRYTARNGGYTRVMKLEPRLGDRAPQAI
gene_mark                   : ::::   ::::: :     : :::   : ::::::  ::::::::: :  
gene_keep                   ====================================================

Kwal0.3             LELVDAPIVGENGTLQRGDLKLWLLCKAAIQNEASQEAYSRLTLQNFAKMLKFRpdphAF
SAKL0D07062g        LELVDNPVIDQNGQFQRGNLKLWLLCKTMLANESAQEPYSDNTLQNFRKIAKFQndfdAF
KLLA0F01793g        LELVDAPVQNEDGQLLKGNLKLWLLTKTcmqqiedagqlapkTVQNLKKMVhnktldq--
gene_mark           ::::: :     :    : :::::: :               : ::  :           
gene_keep           ===========================                                 

Kwal0.3             FVNDVMAVKKFI----------KQEQGLEWDEKHEIGLLEQLESQVLGLSAQPEtsqiQK
SAKL0D07062g        LSKDVLAIRKFI----------KLQEQQPWDEEQELQQLKVLKQKVLSPSQKKN----Hs
KLLA0F01793g        LLHEIGVIRKYIlesqqqnsttEVSELPEFDETLQQQFLAAVKDQIQNHTAEKK----QP
gene_mark                    : :                  ::      :                     
gene_keep                                                                       

Kwal0.3             FVNDVMAVKKFI----------KQEQGLEWDEKHEIGLLEQLESQVLGLSAQPEtsqiQKA---GYTFVNERPGR-
SAKL0D07062g        LSKDVLAIRKFI----------KLQEQQPWDEEQELQQLKVLKQKVLSPSQKKN----HsgstpGFNYVSKRPERl
KLLA0F01793g        LLHEIGVIRKYIlesqqqnsttEVSELPEFDETLQQQFLAAVKDQIQNHTAEKK----QPD---GYVFr-ERPtqg
gene_mark                    : :                  ::      :                         :      ::   
gene_keep                                                                                       



*******************************
*        Argument             *
*******************************
#gap_allow:	0
#min_conserve:	5
#min_flanking:	10
#max_nonconserve:	6
#judge_conserve:	1.0
