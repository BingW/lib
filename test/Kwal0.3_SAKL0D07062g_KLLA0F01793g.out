      3    134

Kwal0.3      MTTGLMRHFS RTKAHRDSMF RNLVSQLFQH GSIVSTHEKC LEASRMAERI ITMAKEIQSR LSLAGDNSKL LRRVVNELAP RYQQRPGGYT RVLKLEPRLG DRAKQSVLEL VDAPIVGENG TLQRGDLKLW LLCK
SAKL0D07062g  MTVGLARHLS RTKPHRDALL KNLVTQLFQH GSITSTHEKC KEASRLAERA ITIAKELQSR LFHSGDNSKV LKKLLQEIAP RCVTRPGGYT RVLHLEPRLG DRARQSVLEL VDNPVIDQNG QFQRGNLKLW LLCK
KLLA0F01793g  MTRGLARHLS RTKPHRDALI RNMATQLLQH GTISSTTPKL KETQRYVERI ITIAKELQSR LYLSGDNSKL LKKLFDEIAP RYTARNGGYT RVMKLEPRLG DRAPQAILEL VDAPVQNEDG QLLKGNLKLW LLTK



Printing out site pattern counts


         3         72  P

Kwal0.3      AAAAAACCDD EEEFFFGGHH IIIIKKKLLL LLLLMMMMMN NNPPQQQQRR RRSSSSSSST TTVVVVVVVW YY
SAKL0D07062g  AAANPSCCDN EEQFLLDGHH AILVHKRFHI KLLVAILLMN NQPPQQTVKK RRAFSSSSTQ TVILLTVVVW CY
KLLA0F01793g  ATVAPSLTDN EPELILNGHT IILVKKPLLI KLMLAILYMD NDNPLQATKR KRAYAQSTTQ TRQFLSAIVW YY



    3    1    1    1    1    1    1    1    4    1    6    1    1    1    1
    1    1    8    3    1    1    2    1    1    1    6    1    1    1    1
    1   12    2    1    1    1    1    1    1    1    2    1    1    3    1
    4    1    1    2    1    1   11    1    1    1    1    4    1    1    1
    5    1    1    1    1    1    1    1    2    1    1    1

AAML (in paml version 4.5, December 2011)  /Users/bingwang/VimWork/lib/test/Kwal0.3_SAKL0D07062g_KLLA0F01793g.aa
Model: REVaa_0 for branches(nrate:74) ns =   3  ls = 134


Frequencies..
                                    A      R      N      D      C      Q      E      G      H      I      L      K      M      F      P      S      T      W      Y      V
Kwal0.3                        0.0597 0.1119 0.0299 0.0373 0.0149 0.0522 0.0597 0.0672 0.0299 0.0373 0.1418 0.0597 0.0373 0.0224 0.0299 0.0746 0.0522 0.0075 0.0149 0.0597
SAKL0D07062g                   0.0597 0.0970 0.0373 0.0373 0.0224 0.0597 0.0522 0.0597 0.0448 0.0373 0.1567 0.0746 0.0075 0.0224 0.0373 0.0597 0.0597 0.0075 0.0075 0.0597
KLLA0F01793g                   0.0672 0.0970 0.0373 0.0448 0.0000 0.0522 0.0522 0.0597 0.0224 0.0522 0.1642 0.0821 0.0224 0.0075 0.0448 0.0448 0.0821 0.0075 0.0299 0.0299

Homogeneity statistic: X2 = 0.11694 G = 0.13177 

Average                         0.0622  0.1020  0.0348  0.0398  0.0124  0.0547  0.0547  0.0622  0.0323  0.0423  0.1542  0.0721  0.0224  0.0174  0.0373  0.0597  0.0647  0.0075  0.0174  0.0498

# constant sites:     78 (58.21%)
ln Lmax (unconstrained) = -520.476455

AA distances (raw proportions of different sites)

Kwal0.3        
SAKL0D07062g     0.2836
KLLA0F01793g     0.3507  0.2910

Time used:  0:00
